colors/onedark.vim

```diff
@@ -297,6 +297,17 @@ call s:h("cssSelectorOp", { "fg": s:purple })
 call s:h("cssSelectorOp2", { "fg": s:purple })
 call s:h("cssTagName", { "fg": s:red })
 
+" Dart
+call s:h("dartTypeDef", { "fg": s:red })
+call s:h("dartClassDecl", { "fg": s:red })
+call s:h("dartInterpolation", { "fg": s:blue })
+highlight link dartLibrary Include
+call s:h("dartSdkClass", { "fg": s:cyan })
+call s:h("dartStorageClass", { "fg": s:dark_red })
+highlight link dartExceptions Exception
+highlight link dartStatement Statement
+call s:h("dartConstant", { "fg": s:dark_yellow })
```

Comment:
`dartConstant` points to `Constant`,  which already has highlighting in onedark.vim